### PR TITLE
Clean up SocketAsyncEventArgs.

### PR DESCRIPTION
**Pull request by pgavlin**

### CODE REVIEW

**Comment by pgavlin** [comment]
@davidsh, @stephentoub PTAL

**Comment by stephentoub** [comment]
LGTM

**Comment by davidsh** [comment]
LGTM

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs [hunk 1]
```diff
@@ -377,86 +383,84 @@ private void InnerStartOperationReceiveMessageFrom()
...
+                _ptrWSAMessageBuffer = Marshal.UnsafeAddrOfPinnedArrayElement(_wsaMessageBuffer, 0);
             }
 
             // Create and pin an appropriately sized control buffer if none already
-            IPAddress ipAddress = (m_SocketAddress.Family == AddressFamily.InterNetworkV6
-                ? m_SocketAddress.GetIPAddress() : null);
-            bool ipv4 = (_currentSocket.AddressFamily == AddressFamily.InterNetwork
-                || (ipAddress != null && ipAddress.IsIPv4MappedToIPv6)); // DualMode
+            IPAddress ipAddress = (_socketAddress.Family == AddressFamily.InterNetworkV6 ? _socketAddress.GetIPAddress() : null);
+            bool ipv4 = (_currentSocket.AddressFamily == AddressFamily.InterNetwork || (ipAddress != null && ipAddress.IsIPv4MappedToIPv6)); // DualMode
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs:398]: I know that we're generally not concerned about line width but this was already <120 chars width. Why expanding these calls on a single line, making it harder to read?

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 2]
```diff
@@ -740,26 +715,30 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
 
                     if (socketError == SocketError.Success)
                     {
-                        m_AcceptSocket = _currentSocket.UpdateAcceptSocket(m_AcceptSocket, _currentSocket._rightEndPoint.Create(remoteSocketAddress));
+                        _acceptSocket = _currentSocket.UpdateAcceptSocket(_acceptSocket, _currentSocket._rightEndPoint.Create(remoteSocketAddress));
 
-                        if (s_LoggingEnabled)
-                            Logging.PrintInfo(Logging.Sockets, m_AcceptSocket,
-          SR.Format(SR.net_log_socket_accepted, m_AcceptSocket.RemoteEndPoint, m_AcceptSocket.LocalEndPoint));
+                        if (s_loggingEnabled)
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:720]: Please add {} to all `if` blocks. (I know this is still acceptable but I find the practice dangerous).

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 3]
```diff
@@ -724,13 +695,17 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
...
-
                     if (bytesTransferred > 0)
                     {
                         // Log and Perf counters.
-                        if (s_LoggingEnabled) LogBuffer(bytesTransferred);
-                        if (Socket.s_perfCountersEnabled) UpdatePerfCounters(bytesTransferred, false);
+                        if (s_loggingEnabled)
+                        {
+                            LogBuffer(bytesTransferred);
+                        }
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:704]: Nit: new line after }

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 4]
```diff
@@ -767,9 +746,8 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
                     // Mark socket connected.
                     if (socketError == SocketError.Success)
                     {
-                        if (s_LoggingEnabled)
-                            Logging.PrintInfo(Logging.Sockets, _currentSocket,
-          SR.Format(SR.net_log_socket_connected, _currentSocket.LocalEndPoint, _currentSocket.RemoteEndPoint));
+                        if (s_loggingEnabled)
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:749]: Please add {} around if blocks.

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 5]
```diff
@@ -777,39 +755,48 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
...
-
                     if (bytesTransferred > 0)
                     {
                         // Log and Perf counters.
-                        if (s_LoggingEnabled) LogBuffer(bytesTransferred);
-                        if (Socket.s_perfCountersEnabled) UpdatePerfCounters(bytesTransferred, false);
+                        if (s_loggingEnabled)
+                        {
+                            LogBuffer(bytesTransferred);
+                        }
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:770]: Nit: new line after }

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 6]
```diff
@@ -777,39 +755,48 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
...
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:785]: Nit: New line after }

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 7]
```diff
@@ -844,34 +836,49 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
...
-
                     if (bytesTransferred > 0)
                     {
                         // Log and Perf counters.
-                        if (s_LoggingEnabled) LogBuffer(bytesTransferred);
-                        if (Socket.s_perfCountersEnabled) UpdatePerfCounters(bytesTransferred, true);
+                        if (s_loggingEnabled)
+                        {
+                            LogBuffer(bytesTransferred);
+                        }
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:845]: Nit: New line after }

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 8]
```diff
@@ -844,34 +836,49 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
...
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:877]: Nit: New line after }

File: src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs [hunk 9]
```diff
@@ -844,34 +836,49 @@ internal void FinishOperationSuccess(SocketError socketError, int bytesTransferr
...
-
                     if (bytesTransferred > 0)
                     {
                         // Log and Perf counters.
-                        if (s_LoggingEnabled) LogSendPacketsBuffers(bytesTransferred);
-                        if (Socket.s_perfCountersEnabled) UpdatePerfCounters(bytesTransferred, true);
+                        if (s_loggingEnabled)
+                        {
+                            LogSendPacketsBuffers(bytesTransferred);
+                        }
```
Comment (CIPop) [inline src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs:860]: Nit: New line after }

**Comment by CIPop** [comment]
A few coding style nitpicks.
LGTM

**Pull request merged**